tests/managers_regress/tests.py

```diff
@@ -201,3 +201,12 @@ def test_regress_3871(self):
             t.render(Context({'related': related})),
             ''.join([force_text(relation.pk)] * 3),
         )
+
+    def test_field_can_be_called_exact(self):
+        # Make sure related managers core filters don't include an
+        # explicit `__exact` lookup that could be interpreted as a
+        # reference to a foreign `exact` field. refs #23940.
+        related = RelatedModel.objects.create(exact=False)
+        relation = related.test_fk.create()
+
```

Comment:
Alright I'll fix those two issues before merging and backporting. Just want to avoid an other PR CI build.